Source/OxyPlot/Series/BarSeries/BarSeries.cs

```diff
@@ -295,14 +344,14 @@ protected override bool IsValid(BarItem item)
         /// </summary>
         /// <param name="rc">The render context</param>
         /// <param name="item">The item.</param>
-        /// <param name="baseValue">The bar item base value.</param>
+        /// <param name="BaseValue">The bar item base value.</param>
         /// <param name="topValue">The bar item top value.</param>
         /// <param name="categoryValue">The bar item category value.</param>
         /// <param name="categoryEndValue">The bar item category end value.</param>
         protected void RenderLabel(
             IRenderContext rc,
             BarItem item,
-            double baseValue,
+            double BaseValue,
```

Comment:
The capital 'B' is probably an accident?

Reply:
Fixed, and rebased on develop